astroquery/mast/core.py

```diff
@@ -1303,7 +1308,7 @@ def enable_s3_hst_dataset(self):
         log.info("Using the S3 HST public dataset")
         log.warning("Your AWS account will be charged for access to the S3 bucket")
         log.info("See Request Pricing in https://aws.amazon.com/s3/pricing/ for details")
-        log.info("If you have not configured boto3, follow the instructions here: " +
+        log.info("If you have not configured boto3, follow the instructions here: " + \
```

Comment:
neither the `+` nor the `\` are needed here

Reply:
Haha, in trying to address a pep8 complaint I made it worse. Fixed now.